js/functional/fibonacci.js

```diff
@@ -6,8 +6,26 @@ import { parseNumber } from "../helper.js";
  * @param {number} sequence
  */
 function generateFibonacciUsingLoop(sequence) {
-  // write your code here
-  return [];
+  const fibonacciArray = [];
```

Comment:
Make a simpler variable name such as "array" or "result" since we know that this variable is used to contain Fibonacci numbers.